bin/wiki_entity_linking/training_set_creator.py

```diff
@@ -27,21 +32,23 @@ def _process_wikipedia_texts(wikipedia_input, wp_to_id, training_output, limit=N
     Read the XML wikipedia data to parse out training data:
     raw text data + positive instances
     """
-    title_regex = re.compile(r'(?<=<title>).*(?=</title>)')
-    id_regex = re.compile(r'(?<=<id>)\d*(?=</id>)')
+    title_regex = re.compile(r"(?<=<title>).*(?=</title>)")
+    id_regex = re.compile(r"(?<=<id>)\d*(?=</id>)")
 
     read_ids = set()
     entityfile_loc = training_output / ENTITY_FILE
-    with open(entityfile_loc, mode="w", encoding='utf8') as entityfile:
+    with open(entityfile_loc, mode="w", encoding="utf8") as entityfile:
```

Comment:
Not a big deal, but we typically want to use pathlib for opening and validating files:

```suggestion
    with entityfile_loc.open("w", encoding="utf8") as entityfile:
```